File: src/main/java/io/dddrive/validation/model/impl/AggregatePartValidationImpl.java
package io.dddrive.validation.model.impl;

import io.dddrive.validation.model.AggregatePartValidation;
import io.dddrive.validation.model.enums.CodeValidationLevel;
import lombok.Builder;
import lombok.Data;

@Data
@Builder
public class AggregatePartValidationImpl implements AggregatePartValidation {

	private Integer seqNr;
	private CodeValidationLevel validationLevel;
	private String validation;

}

File: fm-dddrive/src/main/java/io/dddrive/validation/model/impl/AggregatePartValidationImpl.kt
package io.dddrive.validation.model.impl

import io.dddrive.validation.model.AggregatePartValidation
import io.dddrive.validation.model.enums.CodeValidationLevel

class AggregatePartValidationImpl(
	override val seqNr: Int,
	override val validationLevel: CodeValidationLevel,
	override val message: String,
	override val path: String? = null,
) : AggregatePartValidation
